docs/current_docs/integrations/snippets/openshift.yaml

```diff
@@ -4,15 +4,15 @@ fullnameOverride: ""
 engine:
   image:
     repository: registry.dagger.io/engine
-    tag: v0.10.2
+    tag: latest
```

Comment:
Are we sure that we want `latest`? I'm not hinting that it's wrong, just double-checking that we want to run with whatever is latest when this gets executed.

Benefits of `latest`: auto-updates, less maintenance overhead.
Drawbacks of `latest`: requires explicit update, pipelines may stop working due to breaking changes (e.g. 0.8.0, 0.10.0, etc.)

Benefits of a fixed version: always know what is running in CI, pipelines won't break unless release notes are ignored.
Drawbacks of a fixed version: more maintenance, may fall behind if this is not done regularly.

Either approached are valid. OK to keep as is if it's what we want.

Reply:
Added this info to the page